concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast/00 Display a Toast.md

```diff
@@ -0,0 +1,2 @@
+[note] For simplicity, this tutorial displays a Toast in response to a click on a [Button](/api-reference/10%20UI%20Components/dxButton '/Documentation/ApiReference/UI_Components/dxButton/'). Your application's logic will likely be different, but you can still re-use the code. 
```

Comment:
- Title: Display a Toast (wouldn't it be better to refer to it as a Toast notification? Because just 'Toast' doesn't make much sense)

I'm not sure about this note: 
- `For simplicity` means what? Why add this detail?
- `will likely be different` seems rather obvious since most applications would have different logic.

I'm not sure I understand the point of this note.

Reply:
The point of this note is to show that you can use the same logic even without the button. I think it's neccessary to add this note, so the customers would be sure they can display notifications using different UI.

Reply:
This tutorial displays a Toast notification in response to a click on a [Button](/api-reference/10%20UI%20Components/dxButton '/Documentation/ApiReference/UI_Components/dxButton/'). However, you can re-use the code for different application logic.